ninewinecfg/main.c

```diff
@@ -397,20 +410,21 @@ static void nine_set(BOOL status, BOOL NoOtherArch)
                 remove_file(dst);
         }
 
-        hmod = LoadLibraryExA(fn_nine_dll, NULL, DONT_RESOLVE_DLL_REFERENCES);
+        hmod = LoadLibraryExA(fn_forwarder_dll, NULL, DONT_RESOLVE_DLL_REFERENCES);
         if (hmod)
         {
-            Dl_info info;
+            CHAR buf[MAX_PATH];
+            DWORD len = GetModuleFileNameA(hmod, (LPSTR)buf, sizeof(buf));
 
-            if (dladdr(hmod, &info) && info.dli_fname)
-                create_symlink(info.dli_fname, dst);
+            if (len && len < sizeof(buf))
+                create_symlink(buf, dst);
             else
-                ERR("dladdr failed to get file path\n");
+                ERR("GetModuleFileNameA failed to get file path\n");
```

Comment:
Now that we install the forwarder ourselves, maybe it would make sense to use a relative path for the symlink, so that if the prefix is moved, it doesn't break.